### PR TITLE
Added loans page

### PR DESCRIPTION
**Pull request by kwunkiuma**
image

I'll be having lunch now so no rush to review this 😃

### CODE REVIEW

**Comment by kwunkiuma** [comment]
Forgot to add reviewer 😅

File: Bookish.Web/Views/Home/Loans.cshtml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            </th>
+        </tr>
+        @foreach (var bookLoan in Model.BookLoans)
+        {
+            <tr>
+                <td>@bookLoan.CopyId</td>
+                <td>@bookLoan.Title</td>
+                <td>@bookLoan.Author</td>
+                <td>@bookLoan.Isbn</td>
+                <td>@bookLoan.DueDate.ToShortDateString()</td>
```
Comment (FridaTveit) [inline Bookish.Web/Views/Home/Loans.cshtml:36]: Not necessary to do now, but one extension you could do it to highlight this date in some way if it's overdue 🙂

File: Bookish.Web/Views/Home/Loans.cshtml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                ISBN
+            </th>
+            <th>
+                Due date
+            </th>
+        </tr>
+        @foreach (var bookLoan in Model.BookLoans)
+        {
+            <tr>
+                <td>@bookLoan.CopyId</td>
```
Comment (FridaTveit) [inline Bookish.Web/Views/Home/Loans.cshtml:32]: Each book here should link to the book details page, like on the catalogue 🙂

**Review by FridaTveit — COMMENTED** [review]
One suggestion for an extension and one missed requirement, otherwise looks great! 🙂

**Review by FridaTveit — APPROVED** [review]
Looks great 🙂

**Pull request merged**